packages/realm/react-native/ios/RealmReact/RealmReact.h

```diff
@@ -0,0 +1,36 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2016 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#import <Foundation/Foundation.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef void (^RealmReactEventHandler)(id message);
+
+@interface RealmReact : NSObject
+
+- (void)addListenerForEvent:(NSString *)eventName handler:(RealmReactEventHandler)handler;
```

Comment:
I think this is legacy code from the old react native test infrastructure.  I bet we can safely remove these.